lib/config/options/index.ts

```diff
@@ -1555,6 +1555,14 @@ const options: RenovateOptions[] = [
     allowedValues: ['strict', 'flexible', 'none'],
     default: 'strict',
   },
+  {
+    name: 'prAllowMaintainerEdits',
+    description:
+      'Whether pull requests created should allow for maintainer edits.',
```

Comment:
```suggestion
      'Decides if maintainers can edit Renovate pull requests.',
```

How about something like this?